plugin/pkg/scheduler/api/validation/validation.go

```diff
@@ -34,5 +34,10 @@ func ValidatePolicy(policy schedulerapi.Policy) error {
 		}
 	}
 
+	for _, extender := range policy.ExtenderConfigs {
```

Comment:
Insecure mode is set which skips verification. Look at TLSConfigFor in client package.